docs/API.md

```diff
@@ -395,9 +395,19 @@ This key is added when the module name is `"NAMESERVER"`.
 
 Basic data type: string
 
+#### Deprecated representation
+
+**Deprecated**
 Default database timestamp format: "Y-M-D H:M:S.ms".
 Example: "2017-12-18 07:56:17.156939"
 
+
+#### New representation
```

Comment:
I think "Prescribed representation" or "Recommended representation" would be better. Another possibility:

```
### Deprecated Timestamp

Basic data type: string

**Deprecated**
Default database timestamp format: "Y-M-D H:M:S.ms".
Example: "2017-12-18 07:56:17.156939"

### Timestamp

Basic data type: string

A string representing a date and time using the following ISO 8601 format:
"YYYY-MM-DDThh:mm:ssZ".
Example: "2017-12-18T07:56:17Z"
```

Reply:
I went with your second suggestion.